pytest/tests/mocknet/helpers/load_testing_add_and_delete_helper.py

```
@@ -292,7 +297,8 @@ def main(argv):
     assert delay >= 1
     for i, account in enumerate(test_accounts):
         logger.info(f'Deploying contract for account {i}')
-        account.send_deploy_contract_tx(mocknet.WASM_FILENAME)
+        mocknet_helpers.retry_and_ignore_errors(
```

Do we really want to ignore errors if this fails?

---

Right, but if we fail deploying the contract doesn’t that break other tests?